src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -622,6 +623,16 @@ public bool IsFrozen
             }
         }
 
+        /// <summary>
+        /// Return a value indicating whether this node is in preview state.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsPreview
```

Comment:
@QilongTang can I suggest that we use a different name for this - it seems to conflict with preview bubbles and all the is displaying preview naming that has to do with displaying geometry previews.

What about something like:

`IsProxy` or `IsCommitedToGraph?

Those are bad names, but you get the gist.

I just want to make sure that the different "preview" concepts are not confused.

Reply:
Yeah I noticed that as well and try to rename when they collapse, but I think `IsProxy` could be a good one instead, let me think about it

Reply:
I'm going with this name lol, the fun of naming your property!